Breeze.BreezeServer/BreezeRegistration.cs

```diff
@@ -202,5 +202,33 @@ public Transaction CreateBreezeRegistrationTx(Network network, byte[] data, Mone
 
             return sendTx;
         }
+
+        public bool VerifyCollateral(BreezeConfiguration config, out Money missingFunds)
+        {
+            Network network = Network.StratisMain;
+            if (config.TumblerNetwork == Network.TestNet || config.TumblerNetwork == Network.RegTest)
+            {
+                network = Network.StratisTest;

```

Comment:
Is it valid to have StratisTest here if we are connecting to a StratisRegTest RPC server?

Reply:
I agree that this doesn't look correct but as it already been done that way in `PerformBreezeRegistration` and it works in Testnet and Regtest I've used the same approach here.